projects/kit/components/preview/preview.component.ts

```diff
@@ -113,13 +113,11 @@ export class TuiPreviewComponent {
     }
 
     protected onResize(contentResizeEntries: readonly ResizeObserverEntry[]): void {
-        if (contentResizeEntries.length === 0) {
-            return;
-        }
-
-        const {width, height} = contentResizeEntries[0].contentRect;
+        const rect = contentResizeEntries[0]?.contentRect;
```

Comment:
Let's destruct as `[entry]: ...` in the argument above

Reply:
fixed